.github/workflows/test_cra.yaml

```
@@ -0,0 +1,60 @@
+#         with:
+#           pr: ${{ github.event.issue.pull_request.html_url }}
+#           command: ${{ github.event.comment.body }}
+#           options: ${{ env.OPTIONS }}
+
+#       - name: Code Review Agent action step
+#         if: github.event_name == 'pull_request'
+#         uses: gitbito/codereviewagent@main
+#         with:
+#           pr: ${{ github.event.pull_request.html_url }}
+#           command: review
+#           options: ${{ env.OPTIONS }}
```

yamllint

[error] no new line character at the end of file

(new-line-at-end-of-file)